streams/src/test/java/org/apache/kafka/streams/processor/internals/StreamsPartitionAssignorTest.java

```diff
@@ -226,6 +240,39 @@ public void testSubscription() {
         assertEquals(info.encode(), subscription.userData());
     }
 
+    @Test
+    public void testCooperativeSubscription() {
```

Comment:
Could we comment on the intention of this unit test?

Reply:
Hmm, after some refactoring this looks kind of useless in its current form. I'll try to rewrite it to actually do something